src/riscv/lib/src/interpreter/rv64a.rs

```diff
@@ -223,9 +223,9 @@ mod test {
 
     test_lrsc!(test_lrd_scd, run_lrd, run_scd, 8, u64);
 
-    test_lrsc!(test_lrd_scw, run_lrd, run_scw, 8, u32);
-
-    test_lrsc!(test_lrw_scd, run_lrw, run_scd, 8, u32);
+    // TODO: RV-663 - reimplement tests once LR.D and SC.D are implemented in ICB.
+    // test_lrsc!(test_lrd_scw, run_lrd, run_scw, 8, u32);
+    // test_lrsc!(test_lrw_scd, run_lrw, run_scd, 8, u32);
```

Comment:
I feel like this is just a matter of qualifying the identifier for the `run_scw` and `run_lrw` methods. You could probably just re-use `test_atomic_loadstore` for this. I think this way you don't need to disable these tests at all.